config.yaml

```diff
@@ -0,0 +1,5 @@
+options:
```

Comment:
I agree, but it will be handled in kratos-ui-operator interface, since this url is needed by both kratos and hydra. So there will be another PR once implemented on kratos-ui side